### PR TITLE
Fixes errors in doc header and operator install command

### PR DESCRIPTION
**Pull request by zuston**
### What changes were proposed in this pull request?
Fixes errors in doc header and operator install command

### Why are the changes needed?
Fixes errors in doc header and operator install command

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
Don't need

### CODE REVIEW

File: docs/uniffle-migration-guide.md [hunk 1]
```diff
@@ -10,9 +10,7 @@ this work for additional information regarding copyright ownership.
 The ASF licenses this file to You under the Apache License, Version 2.0
 (the "License"); you may not use this file except in compliance with
 the License.  You may obtain a copy of the License at
-
-      http://www.apache.org/licenses/LICENSE-2.0
-
+http://www.apache.org/licenses/LICENSE-2.0
```
Comment (kaijchen) [inline docs/uniffle-migration-guide.md:13]: Why do we need to change this? Looks like other docs also uses this format.

https://github.com/apache/incubator-uniffle/blob/master/docs/server_guide.md?plain=1

Comment (zuston) [reply docs/uniffle-migration-guide.md:13]: This page is error. https://github.com/apache/incubator-uniffle/blob/master/docs/uniffle-migration-guide.md

Comment (kaijchen) [reply docs/uniffle-migration-guide.md:13]: Can we just move the lisense into comments instead of yaml header?

Comment (zuston) [reply docs/uniffle-migration-guide.md]: Comments? I don't get your point. I have uploaded latest commit and fixed its style.

**Review by kaijchen — APPROVED** [review]
LGTM, thanks @zuston for the fix.

**Pull request merged**